hack/golangci.yaml.in

```
@@ -148,6 +148,9 @@ linters:
     - revive
     - staticcheck
     - stylecheck
+    {{- if .Strict}}
```

This ends up only in hack/golangci-strict.yaml, but not golangci-hints.yaml. Each config should include everything from the previous one, so:

```suggestion
    {{- if not .Base}}
```